tests/openssl-rand-provider/env.c

```diff
@@ -0,0 +1,99 @@
+/*
```

Comment:
Note to self: this code duplication of env.c is not good, I need to think about how to handle it. This comment, however, shall not hold up any merging.